Ethics_Law_Professionalism.md

```diff
@@ -1,92 +1,94 @@
+- [How the 'Original Internet Godfather' walked away from his cybercrime past](https://theconversation.com/how-the-original-internet-godfather-walked-away-from-his-cybercrime-past-interview-88822)
+- [Call us immediately if your child uses Kali Linux says West Mids Police](https://www.theregister.com/2020/02/14/silly_police_infosec_parental_advice_poster/)
+- [Jeffrey Paul: Your Computer isn't yours](https://sneak.berlin/20201112/your-computer-isnt-yours/)
+- [Does Apple really log every app you run?](https://blog.jacopo.io/en/post/apple-ocsp/)
+- [I asked Tinder for my data. It sent me 800 pages of my deepest, darkest secrets.](https://www.theguardian.com/technology/2017/sep/26/tinder-personal-data-dating-app-messages-hacked-sold)
+- [A hacker got all my texts for $16](https://www.vice.com/en/article/y3g8wb/hacker-got-my-texts-16-dollars-sakari-netnumber)
+- [Your period tracking app could be sharing intimate details with all of facebook](https://www.theguardian.com/world/commentisfree/2019/sep/14/your-period-tracking-app-could-be-sharing-intimate-details-with-all-of-facebook)
+- [Shaping the Future of Connected Policing](https://www.youtube.com/watch?v=24FaeP_Zv_8)
+- [Enabling Digital Policing: Computacenter and VMware](https://www.youtube.com/watch?v=wwJyKUv3rcs)
+- [NSA Files Decoded: Snowden's surveillance revelations explained](https://www.theguardian.com/world/interactive/2013/nov/01/snowden-nsa-files-surveillance-revelations-decoded#section/1)
+- [Surveillance Technologies under review](https://www.seattle.gov/tech/initiatives/privacy/surveillance-technologies)
+- [I don't know how to count that low](https://acesounderglass.com/2021/10/20/i-dont-know-how-to-count-that-low/) - Dismissing smaller problems doesn't mean they don't exist or that they don't need solving.
+- [Hackers turning to exotic programming languages](https://thehackernews.com/2021/07/hackers-turning-to-exotic-programming.html?m=1)
+- [Don't talk to Police](https://www.youtube.com/watch?v=d-7o9xYp7eE) - An American Law School lecture with a police officer in attendance exploring why it is never a good idea to talk to police.
+- [Right to be offended does not exist - High Court Judge](https://www.dailymail.co.uk/news/article-7710009/Right-offended-does-not-exist-says-High-Court-judge.html) - High court Judge on why the right to be offended does not exist.
 
 ## Professionalism & Standards <a name="standards"></a>
-1. [W3](https://www.w3.org/standards/) - Standards for web design and applications, devices, architecture, XML, web services and other technology.
+- [W3](https://www.w3.org/standards/) - Standards for web design and applications, devices, architecture, XML, web services and other technology.
 guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/)
```

Comment:
is this part of a link??

Reply:
https://github.com/FreesideHull/StudentResources/blob/main/Ethics_Law_Professionalism.md#professionalism--standards- - This is within the repo currently so im not sure 😆

Reply:
I'd say remove the `guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/)` part